processing/data_collection/gazette/spiders/mg_itauna.py

```diff
@@ -0,0 +1,36 @@
+import datetime as dt
+from dateparser import parse
+from scrapy import Request
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class MgItaunaSpider(BaseGazetteSpider):
+    TERRITORY_ID = "3133808"
+    name = "mg_itauna"
+    allowed_domains = ["itauna.mg.gov.br"]
+    start_urls = ["https://www.itauna.mg.gov.br/portal/diario-oficial/"]
+
+    def parse(self, response):
+        url = self.start_urls[0] + "{i}/0/0/0/0/"
+        last_page = int(response.xpath("//select/option/text()")[-1].extract())
+        for i in range(1, 1 + last_page):
```

Comment:
**suggestion**: As I mentioned before, a clear variable name would make the code more readable.

Compare this:
```
for i in range(1, 1 + last_page):
    yield Request(url.format(i=i, callback=self.parse_editions_page)
```

With this:
```
for page in range(1, 1 + last_page):
    yield Request(url.format(page=page), callback=self.parse_editions_page)
```

In my opinion the second is much more readable.